debian/postinst

```diff
@@ -22,6 +22,8 @@ case $1 in
 	systemctl enable systemd-networkd.service
 
 	systemctl disable walinuxagent.service
+
+	systemctl reload delphix-platform.service
```

Comment:
Should this happen before we re-enable the service at line 21?

Reply:
I don't think so, because enabling the service doesn't cause the service to start running. It just makes it eligible to run the next time the system boots up.

So, we install the package, enable the service, then ensure the "ansible-done" file is removed, then wait for the service to be started by something else (e.g. the upgrade scripts, or a reboot, or whatever).

Does this make sense?

Reply:
Yes, thanks for clarifying.